package.json

```diff
@@ -84,6 +83,6 @@
     "uglify-js": "2.6.4",
     "underscore": "1.8.3",
     "watch": "0.19.1",
-    "webpack": "1.14.0"
+    "webpack": "2.2.0"
```

Comment:
If this version is not visible on npm, is cause it still behind the beta flag. Reason for this choice by the wp team is to speed up writing documentation. They will remove the beta flag once documentation is fully complete. But webpack itself is totally out of beta and stable